### ISSUE TITLE
End-user documentation for migration from IMAP

### ISSUE THREAD

**Issue by nidico** [zammad/zammad#575]
I'm missing some end-user documentation on best practice workflows about how to start using Zammad. (I am only aware of the docs at http://docs.zammad.org/en/latest/)

In our case, we're using shared IMAP folders for support emails until now, thus we now have a full inbox with some subfolders. What would be a good way to get started? Is it a stupid idea trying to import existing email threads in the first place and is it simply best to start with an empty inbox?

**Comment by imbehind** [comment]
What happens is as Zammad starts to pull from IMAP it creates users and if I check a particular user perspective I can see the email threads/tickets, but I cannot see the tickets from my agent's account.

Also the sentence "Zammad will move it all from your mailbox into Zammad." is pretty frightening. "MOVE"?

Please document this a bit better.

**Comment by imbehind** [comment]
Please add the following options for migrating/importing IMAP/email:

1. Enable the admin to choose between COPY or MOVE (default is MOVE (!!!)) emails from IMAP
2. Enable the admin to enter a timestamp to enable functionality of importing from certain date/time onwards
3. Enable the admin to in-place configure (turn on/off) notifications and similar behaviour.

At least document the procedure in detail.

**Comment by thorsteneckel** [comment]
In the meantime we introduced our three documentations:

https://github.com/zammad/zammad-documentation
https://github.com/zammad/zammad-user-documentation
https://github.com/zammad/zammad-admin-documentation

We'd love to receive a pull request if things are not clear - or at least an issue describing what you're missing. If you're unsure make sure to check out the [community board](http://community.zammad.org/). Closing now.
